app/build.gradle

```diff
@@ -50,30 +50,65 @@ android {
         buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
         buildConfigField "boolean", "SHOW_CONTROL_NUMBER_MENU", 'false'
         buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
-        resValue "string", "app_name", "openIMIS Policies"
```

Comment:
Why did you removed the app name from configuration? We are using this to make sure every build variant has another name and, if installed on same smartphone, can differentiate between apps. It also help to know which version you are running. Please add this and adapt the names for the new BV. The app_name is removed from the translation files. 
As it gives an error when having this in string.xml and build.graddle, is there a solution to replace the app name in string.xml when this is present in build.graddle?
https://stackoverflow.com/questions/24785270/how-to-change-app-name-per-gradle-build-type/36455983